lib/double/double.dart

```diff
@@ -51,4 +51,22 @@ extension Double_ on double {
   Tween<double> tweenTo(double end) {
     return Tween<double>(begin: this, end: end);
   }
+
+  /// Returns a [bool] if [this] value is between [first, second] values, first
+  /// could be greater then second, in this case compared (second, first) interval
+  ///
+  /// Example:
+  /// ```dart
+  /// 100.between(99, 202) // true;
+  /// 100.between(202, 99) // true;
+  /// ```
+  bool between(double first, double second) {
+    ArgumentError.checkNotNull(first, "first");
+    ArgumentError.checkNotNull(second, "second");
+    if (first == second)
```

Comment:
I wouldn't check this since where you are using an inclusive between.
I'll expect `1.between(1,1)` to be `true`.

Reply:
Fixed logic and tests.
Now the `1.between(1,1)` is `true`.